lib/src/multires_stochastic_texture_synthesis.rs

```diff
@@ -909,25 +891,14 @@ impl Generator {
 
                                 // 3.2 get pattern for guide map if we have them
                                 let (my_cost, guide_cost) = if let Some(ref in_guides) = guides {
-                                    // populate guidance patterns for candidates
-                                    for (cand_i, cand) in candidates.iter().enumerate() {
-                                        k_neighs_to_color_pattern(
-                                            &cand.k_neighs,
-                                            image::Rgba([0, 0, 0, 255]),
-                                            &in_guides.example_guides,
-                                            &mut candidates_guide_patterns[cand_i],
-                                            false,
-                                        );
-
-                                        //get example pattern to compare to
-                                        k_neighs_to_color_pattern(
```

Comment:
I'm not sure why this was in the candidates loop. It seemed to not matter so I moved it out